custom_components/ha-glocaltokens/api.py

```diff
@@ -85,15 +87,16 @@ def get_alarms_and_timers_from(self, device, endpoint):
 
     def get_google_devices_information(self):
         _LOGGER.debug("Fetching data...")
-        devices = self._client.get_google_devices_json()
+        devices = json.loads(self._client.get_google_devices_json())
 
         for device in devices:
             # To avoid keyerror's
             timers = []
             alarms = []
-
+            
             device[DEVICE_IP] = "192.168.0.205"  # For testing purpose only
             device[DEVICE_PORT] = PORT  # For testing purpose only
```

Comment:
And then `device.port` here

Reply:
Some thing went wrong i removed this in this pull. I will investigate and make a new commit